### PR TITLE
Adapt a new structure of module arguments

### PR DESCRIPTION
**Pull request by revit13**
This PR implements part of issue #33
It replaces the "copy" with "assets" in the module resources.

Handling the app-uuid will be done in a separate issue dedicated to improving logging.

### CODE REVIEW

File: hello-world-module/values.sample.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ image:
 imagePullSecrets: []
 
 # Action (copy/read)
-copy:
+assets:
   # Copy source
   source:
```
Comment (shlomitk1) [inline hello-world-module/values.sample.yaml:20]: `- source:`

File: hello-world-module/values.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ image:
   pullPolicy: IfNotPresent
 imagePullSecrets: []
 # Action (copy/read)
-copy:
+assets:
   # Copy source
   source:
```
Comment (shlomitk1) [inline hello-world-module/values.yaml:16]: There is a mismatch between declared values and the sent values. Manager sends an `assets `array, not a single structure.

File: hello-world-module/values.sample.yaml (hunk 1 above)
Comment (revit13) [reply hello-world-module/values.sample.yaml:20]: @shlomitk1 Fixed. Thanks

File: hello-world-module/values.yaml (hunk 2 above)
Comment (revit13) [reply hello-world-module/values.yaml:16]: Fixed. Thanks

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**